test/web-platform-tests/tools/update-to-run.js

```diff
@@ -0,0 +1,97 @@
+// Need?
+// using yaml AST?
+
+"use strict";
+const readline = require("readline");
+const path = require("path");
+const fs = require("fs");
+const os = require("os");
+const jsYAML = require("js-yaml");
+const Mocha = require("mocha");
+
+const mocha = new Mocha({});
+
+const toRunFilename = resolveWpt("to-run.yaml");
+const toRunString = fs.readFileSync(toRunFilename, { encoding: "utf-8" });
+const toRunDocs = jsYAML.safeLoadAll(toRunString, { filename: toRunFilename });
```

Comment:
The second argument to [`loadAll`](https://github.com/nodeca/js-yaml#loadall-string--iterator---options-) and [`safeLoadAll`](https://github.com/nodeca/js-yaml#safeloadall-string--iterator--options-) is an optional callback function.

The third is the options arguments.

```suggestion
const toRunDocs = jsYAML.safeLoadAll(toRunString, null, { filename: toRunFilename });
```

Reply:
I dive into jsyaml code find `safeLoadAll` seems has a bug caused this fix can't work. Change to the code below can fix.

```js
const toRunDocs = jsYAML.loadAll(toRunString, null, { filename: toRunFilename,
  schema: jsYAML.DEFAULT_SAFE_SCHEMA });
```

Reply:
I thought the commit suggestion is just to submit my suggestion and then click it...